mlrun/__main__.py

```diff
@@ -102,7 +102,9 @@ def main():
 )
 @click.option("--uid", help="unique run ID")
 @click.option("--name", help="run name")
-@click.option("--workflow", help="workflow name/id")
+@click.option(
+    "--workflow", help="sets the run pipeline labels to match given workflow name/id"
```

Comment:
I think the word "run" is superfluous, yes?

Reply:
not really as what it actually do is spinning a run that its label matches the given value of `--workflow`